packages/ui-extensions/docs/surfaces/checkout/reference/examples/ui-components/progress-determinate-state.example.ts

```diff
@@ -7,39 +7,50 @@ import {
   TextBlock,
 } from '@shopify/ui-extensions/checkout';
 
-export default extension('purchase.checkout.block.render', (root) => {
-  const label = 'Queue process';
+export default extension(
```

Comment:
@oleksandroliynyk Should we keep this change as part of this release to `unstable`, or exclude it? thanks

Link to original PR: https://github.com/Shopify/checkout-web/pull/34322

Also, can we keep new image file? `packages/ui-extensions/docs/surfaces/checkout/screenshots/progress-determinate.png`

Reply:
My assumption here is that we can include this, but I will defer to the author here.